workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-checkout-step/src/main/java/com/liferay/sample/SampleSpringBootApplication.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.sample;
+
+import com.liferay.client.extension.util.spring.boot.ClientExtensionUtilSpringBootComponentScan;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+
+/**
+ * @author Raymond Augé
+ * @author Gregory Amerson
+ * @author Brian Wing Shun Chan
+ */
+@Import(ClientExtensionUtilSpringBootComponentScan.class)
+@SpringBootApplication
+public class SampleSpringBootApplication {
```

Comment:
Could be the wrong name, we still don't have an answer by BChan, but we used something like  CustomCheckoutStepApplication (or CustomCheckoutStepSpringBootApplication).

Reply:
good point, did he answer in the meanwhile?